SPECS/kernel/CVE-2024-46794.nopatch

```diff
@@ -0,0 +1,4 @@
+CVE-2024-46794 - Introducing commit(s) not present in LTS - (generated by autopatch tool)
+upstream fix commit: b6fb565a2d15277896583d471b21bc14a0c99661 
+upstream introducing commit: 31d58c4e557d46fa7f8557714250fb6f89c941ae 
```

Comment:
Confirmed. Did not find this commit https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?id=31d58c4e557d in the stable kernel tree